src/main/java/pl/km/exercise291/user/dto/UserDto.java

```diff
@@ -0,0 +1,49 @@
+package pl.km.exercise291.user.dto;
+
+public class UserDto {
+    private String login;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private boolean isAdmin;
```

Comment:
Zamiast ustawiać to pole w dto, skorzystaj z atrybutu sec:authorize w thymeleafie, pokazane w code review